### PR TITLE
fix decimal place on swap table

### PR DESCRIPTION
**Pull request by robel91**
fix_decimal_test
fix_decimal_run
# Description

_Please include a summary of changes and related issue (if any)._

# Tests

- [ ] Please provide evidence of your successful `dbt run` / `dbt test` here
- [ ] Any comparison between `prod` and `dev` for any schema change

# Checklist
- [ ] Follow [dbt style guide](https://github.com/dbt-labs/corp/blob/main/dbt_style_guide.md)
- [ ] Run `git merge main` to pull any changes from remote into your branch prior to merge.
- [ ] Tag the person(s) responsible for reviewing proposed changes
- [ ] **IMPORTANT** Note for deployment if a `full-refresh` is needed for any model this PR impacts.
     - If any schema is changing, whether it be a table or a view, we will need to run a `full-refresh` in prod to avoid errors with the scheduled refresh jobs.
     - The PR can be reviewed, but should not be merged unless by someone with prod access.
         - Tag `forgxyz` if a full-refresh will be required on prod.

### CODE REVIEW

File: models/silver/silver__dex_swaps.sql [hunk 1]
```diff
@@ -26,13 +26,13 @@ with
                 attributes:coin_received:currency_0::string,
                 attributes:coin_received:currency::string
             ) as from_currency,
-            round(from_amount / pow(10, 6)) as from_decimal,
```
Comment (geewynn) [inline models/silver/silver__dex_swaps.sql:29]: Maybe use a fixed value for the decimals, 6 until we have the  token labels table.

File: models/silver/silver__dex_swaps.sql [hunk 2]
```diff
@@ -26,13 +26,13 @@ with
...
+            from_amount / pow(10, 6) as from_decimal,
             coalesce(
                 attributes:coin_received:amount_1::integer,
                 attributes:wasm:return_amount::integer
             ) as to_amount,
             attributes:wasm:ask_asset::string as to_currency,
-            round(to_amount / pow(10, 6)) as to_decimal,
```
Comment (geewynn) [inline models/silver/silver__dex_swaps.sql:35]: same here.

File: models/silver/silver__dex_swaps.sql (hunk 1 above)
Comment (forgxyz) [reply models/silver/silver__dex_swaps.sql:29]: Yes, the `x_decimal` column is supposed to be the number of decimals the analyst uses to adjust the raw amount to a decimal-adjusted amount. For example, on ethereum, if you look at the contract for [Dai on Etherscan](https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f) the **Profile Summary** lists 18 as the decimal. This is the decimal precision of the raw number we see on chain.

This column is not adjusted amount.

**Review by forgxyz — CHANGES_REQUESTED** [review]

**Comment by liltims77** [comment]
@forgxyz  you mean round(from_amount / pow(10, adjusted_amount - raw_amount)) as from_decimal?

**Comment by forgxyz** [comment]
No, I'll tag you in Discord for context

**Review by geewynn — APPROVED** [review]

**Review by forgxyz — APPROVED** [review]

**Pull request merged**